src/button/button.scss

```diff
@@ -50,6 +50,23 @@ $iui-button-padding-large: $iui-xs * 6;
   &:disabled {
     @include iui-button-disabled;
   }
+
+  &.iui-active {
+    // Shows active stripe only if --_active-stripe-inset is set (from outside)
+    &::after {
+      content: '';
+      position: absolute;
+      inset: var(--_active-stripe-inset);
```

Comment:
Do you have some resources to read about this approach? Is it a new way of writing css or a new way to hack around complicated selectors?
Just I have the concern that it might start to get complicated to catch all the parts if we gonna introduce this in more places...